doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,300 @@
+
+.. code-block:: text
+
+    subsection insertion info
+      set insertion method                               = volume
+      set inserted number of particles at each time step = 100000
+      set insertion frequency                            = 2000
+      set insertion box minimum x                        = -0.075
+      set insertion box minimum y                        = -0.075
+      set insertion box minimum z                        = 0
+      set insertion box maximum x                        = 0.075
+      set insertion box maximum y                        = 0.075
+      set insertion box maximum z                        = 0.07
+      set insertion distance threshold                   = 1.05
+      set insertion maximum offset                       = 0.3
+      set insertion prn seed                             = 19
+    end
+
+Floating Walls
+~~~~~~~~~~~~~~~~~~~
+
+When we pack the cylinder with particles, we need to keep them inside and prevent them from falling through the small inlet channel. To do so, we place the stopper at the bottom of the cylinder, which is :math:`z = 0` plain, as in:
+
+.. code-block:: text
+
+    subsection floating walls
+      set number of floating walls = 1
+      subsection wall 0
+        subsection point on wall
+          set x = 0
+          set y = 0
+          set z = 0
+        end
+        subsection normal vector
+          set nx = 0
+          set ny = 0
+          set nz = 1
+        end
+        set start time = 0
+        set end time   = 50
+      end
+    end
+
+---------------------------
+Running the DEM Simulation
+---------------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``lethe-particles`` executable is within your path, the simulation can be launched on a single processor by typing:
+
+.. code-block:: text
+  :class: copy-button
+
+  lethe-particles dem-packing-in-spouted-bed.prm
+
+or in parallel (where 8 represents the number of processors)
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 8 lethe-particles dem-packing-in-spouted-bed.prm
+
+After the particles have been packed inside the square bed, we can move on to the fluid-particles simulation.
+
+
+-----------------------
+CFD-DEM Parameter File
+-----------------------
+
+The CFD simulation is to be carried out using the packed bed simulated in the previous step. Here we will focus on the modified section as well.
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation is run for 5 s with a time step of 0.001 s. The time scheme and setting for output is shown as follows.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method               = bdf1
+      set number mesh adapt    = 0
+      set output name          = result_
```

Comment:
This is a non-standard output name and the _ is not necessary, you could remove that parameter.